apps/mobile/src/features/stacks-tx-signer/stacks-tx-signer.tsx

```diff
@@ -0,0 +1,157 @@
+import { useState } from 'react';
+
+import { formatBalance } from '@/components/balance/balance';
+import { useStxMarketDataQuery } from '@/queries/market-data/stx-market-data.query';
+import { useAccountByIndex } from '@/store/accounts/accounts.read';
+import { useStacksSigners } from '@/store/keychains/stacks/stacks-keychains.read';
+import { useNetworkPreferenceStacksNetwork } from '@/store/settings/settings.read';
+import { destructAccountIdentifier } from '@/store/utils';
+import { t } from '@lingui/macro';
+import {
+  StacksTransaction,
+  TokenTransferPayload,
+  addressToString,
+  deserializeTransaction,
+} from '@stacks/transactions';
+
+import { type CryptoCurrency, FeeTypes, MarketData } from '@leather.io/models';
+import { Approver, Box, Text } from '@leather.io/ui/native';
+import {
+  baseCurrencyAmountInQuoteWithFallback,
+  convertToMoneyTypeWithDefaultOfZero,
+} from '@leather.io/utils';
+
+import { ApproverAccountCard } from '../approver/components/approver-account-card';
+import { ApproverButtons } from '../approver/components/approver-buttons';
+import { OutcomeAddressesCard } from '../approver/components/outcome-addresses-card';
+import { StacksFeeCard } from '../approver/components/stacks-fee-card';
+import { StacksOutcome } from '../approver/components/stacks-outcome';
+import { ApproverState } from '../approver/utils';
+import { useBroadcastStxTransaction } from './use-broadcast-stx-transaction';
+
+function formReviewTxSummary({
+  tx,
+  symbol = 'STX',
```

Comment:
Not sure on setting default here, seems like a behaviour that might cause bugs. E.g. missing symbol and not realising because there's no type error. Could we just be explicit every time?